capstone-project/src/main/javaTests/com/google/sps/data/MarkerTest.java

```diff
@@ -0,0 +1,100 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public final class MarkerTest {
+
+    private static Marker marker;
+    private static Entity markerEntity;
+    private static final LocalServiceTestHelper helper =
+            new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+    @Before
+    public void setUp() {
+        marker = new Marker.Builder()
+                .setId(1111)
+                .setLat(1)
+                .setLng(1)
+                .setAnimal("animal")
+                .setReporter("reporter")
+                .setDescription("description")
+                .setUserId(Optional.of("userId"))
+                .setBlobKey("blobKey")
+                .build();
+        helper.setUp();
+        markerEntity = new Entity("Marker");
```

Comment:
Try using:

new Entity("marker", 1111)

Then you don't need to sky on the entity